src/Volo.Abp.Core/Volo/Abp/Check.cs

```diff
@@ -40,5 +40,15 @@ public static ICollection<T> NotNullOrEmpty<T>(ICollection<T> value, [InvokerPar
 
             return value;
         }
+        public static string IsNumeric(
+            string value,
+            [InvokerParameterName][NotNull] string parameterName)
```

Comment:
* You request a `[NotNull]` value but not checking it. I mean this method doesn't throw an exception if parameterName is null.
* To be consistent with the other methods, `value` is a better parameter name.
* Add `[ContractAnnotation("value:null => halt")]` just like the other methods.
* `IsNumeric` naming is not clear enough to understand if it accepts floating point numbers (float, double...) or only accepts whole numbers (int/long). Maybe it can get an optional parameter to allow floating point numbers, which is false by default - so only allows integers by default.